main/modes/fighter/mode_fighter.c

```diff
@@ -312,7 +312,7 @@ static const song_t f1hit =
 };
 
 // Simple sound effect when fighter 2 is hit
-song_t f2hit =
+const song_t f2hit =
```

Comment:
I think you missed `f1hit`

Reply:
both appear to have been missed - added them!